ui/raidboss/popup-text.ts

```diff
@@ -316,17 +316,22 @@ class TriggerOutputProxy {
 
   getReplacement(
       // Can't use optional modifier for this arg since the others aren't optional
-      template: { [lang: string]: unknown } | undefined,
+      template: { [lang: string]: unknown } | string | undefined,
       params: TriggerParams,
       name: string,
       id: string): string | undefined {
     if (!template)
       return;
 
-    const value = template[this.displayLang] ?? template['en'];
+    let value: unknown;
```

Comment:
I this this could be shortcuted with `template[this.displayLang] ?? template['en'] ?? template`, but typechecking deny it.